uploads/assignment/run.py

```diff
@@ -209,7 +209,7 @@ def submit_assignment(self):
 
             result_json = r.json()
             sys.exit("ERROR: " + result_json['message'])
-        elif r.status_code == 400:
+        elif r.status_code == 400 or r.status_code == 404:
```

Comment:
I believe `result['status']` is different from `r.status_code`. Should probably be using just one of those. But it seems fine as it is.

Reply:
Yes `r.status_code` is the right method (HTTP Headers). In the beginning I used `result['status']` (JSON) which wasn't right :) 

`result['status']` is not received when error is generated from server for example Error 500. `r.status_code` is simpler when handling status code in run.py

@recluze